src/Host/Client/Impl/Host/RHostBrokerConnector.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.R.Interpreters;
+
+namespace Microsoft.R.Host.Client.Host {
+    public sealed class RHostBrokerConnector : IRHostBrokerConnector {
```

Comment:
It isn't passed to the session operations, it is passed to the session provider operations. Session gets its connector in constructor and uses internally later. 

---
In reply to: [73033191](https://github.com/Microsoft/RTVS/pull/2133#discussion_r73033191) [](ancestors = 73033191)